kolibri/plugins/management/assets/src/vue/data-page/index.vue

```diff
@@ -53,6 +48,28 @@
   const coreApp = require('kolibri');
 
   module.exports = {
+    $trNameSpace: 'manageData',
+    $trs: {
+      // Headings/subHeadings
+      pageHeading: 'Export Usage Data',
+      pageSubHeading: 'Download CSV (comma-separated value) files containing information about' +
```

Comment:
it's fine - there's no silver bullet here, and there are many solutions:

**Basic concatenation:**

```javascript
      detailsInfo: 'When a user views a piece of content, we record how long they spend and the ' +
        'progress they make. Each row in this file records a single visit a user made to a ' +
        'specific piece of content. This includes anonymous usage, when no user is logged in.',
```

**Join concatenation:**

This was faster than `+` many years ago, but these days it's not true for modern JS.

```javascript
      detailsInfo: ['When a user views a piece of content, we record how long they spend and the ',
        'progress they make. Each row in this file records a single visit a user made to a ',
        'specific piece of content. This includes anonymous usage, when no user is logged in.'].join(''),
```

**ES6 multi-line templates:**

Note that the extra indentation spaces are included in the output. This might work well for this case because extra whitespace should be ignored in the HTML output.

```javascript
      detailsInfo: `When a user views a piece of content, we record how long they spend and the 
        progress they make. Each row in this file records a single visit a user made to a
        specific piece of content. This includes anonymous usage, when no user is logged in.`,
```


**ES6 de-dented multi-line templates:**

Not pretty, and may not pass lint...

```javascript
      detailsInfo: `When a user views a piece of content, we record how long they spend and the
        progress they make. Each row in this file records a single visit a user made to a
        specific piece of content. This includes anonymous usage, when no user is logged in.`,
```


**ES6 multi-line templates with de-dent pre-processor:**

Just came across this [babel plugin](https://github.com/MartinKolarik/babel-plugin-dedent). However now we've switched to [Buble](https://buble.surge.sh/guide/) which doesn't support plugins so it's not an option.

```javascript
      detailsInfo: dedent`When a user views a piece of content, we record how long they spend and the
progress they make. Each row in this file records a single visit a user made to a
specific piece of content. This includes anonymous usage, when no user is logged in.`,
```